src/main/webapp/app/overview/student-questions/student-question-row.component.html

```diff
@@ -49,12 +52,19 @@ <h6 class="text-black-50 question-subtitle">{{ studentQuestion.author.name }} -
                 </div>
             </div>
         </div>
+        <!-- question in edit mode -->
         <div *ngIf="isEditMode">
             <div class="row mb-1 mt-2">
                 <div class="col-12">
                     <div class="form-group">
                         <label class="form-control-label">{{ 'artemisApp.courseOverview.exerciseDetails.faq.newQuestionLabel' | translate }}</label>
-                        <textarea [(ngModel)]="studentQuestionText" class="form-control" name="answer"></textarea>
+                        <jhi-markdown-editor
+                            class="markdown-editor background-editor-color"
+                            [(markdown)]="studentQuestionText"
+                            [editorMode]="EditorMode.LATEX"
+                            [enableResize]="true"
+                            [metaCommands]="[]"
```

Comment:
Could `[]` be a default value for the Input that doesn't need to be passed in explicitly?

Reply:
I needed to do it that way for now in order to keep the fullscreen option in every other usage of the markdown editor and disable it here, but I will change this when refactoring the whole editor in a follow-up PR.